scripts/do_all_things.py

```diff
@@ -17,9 +16,6 @@ def main(versions, no_docs, source, skip_install, use_yarn):
     else:
         build_docs.main(["latest"], skip_install, use_yarn)
 
-    print("Copying build output to ../published_versions. Use that directory to publish the site.")
```

Comment:
I moved this into `build_docs.py` since a lot of the copy and file management tasks were already taken care of in that script.